docs/.vuepress/styles/palette.styl

```
@@ -1,2 +1,2 @@
 // https://vuepress.vuejs.org/default-theme-config/#simple-css-override
-$accentColor = #b10438
+//$accentColor = #b10438
```

why have you commented out this color?

```suggestion
$accentColor = #b10438
```

---

Not sure. I think I was testing something and forgot to reset it.